frontend/tests/index.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@playwright/test"
+import { mockIndex } from "./request_mocks.js"
+
+test("test go to movie", async ({ page }) => {
+  await mockIndex(page)
+  await page.route("http://api.localhost/search/harry%20p?c=DK&limit=30", route =>
+    route.fulfill({
+      status: 200,
+      path: "tests/test_data/harry_p_hits.json",
+    })
+  )
+
+  await page.goto("/")
+  const searchbarPlaceholder = page.locator("#input-field")
+  await expect(searchbarPlaceholder).toHaveText("")
+  await searchbarPlaceholder.fill("Harry p")
+  const altTagExists = page.locator('[alt="Harry Potter and the Goblet of Fire"]')
+  const altTagDoesNotExist = page.locator('[alt="Godzilla vs. Kong"]')
+  await expect(altTagExists).toBeVisible()
+  await expect(altTagDoesNotExist).not.toBeVisible()
```

Comment:
Maybe a longshot, but TMDB still holds all images and we just use their URL. So if they decide to change the URL, the test would be bricked right?

Reply:
I might just be misunderstandinh the `not.toBeVisible()`